R/build_part_body.R

```diff
@@ -3,34 +3,34 @@
 #' @param subject an anonymous variable (for instance, and by default, "?subject") or item (for instance "wd:Q456"))
 #' @param verb the property (for instance "wdt:P190")
 #' @param object an anonymous variable (for instance, and by default, "?object") or item (for instance "wd:Q456"))
-#' @param required whether the existence of a value for the triplet is required or not (defaults to TRUE).
-#'   If set to FALSE, then other triplets in the query are returned even if this particular triplet is missing)
-#' @param within_box if provided, north-west and south-east coordinates of bounding box for the triplet query.
-#' @param within_distance if provided, north-west and south-east coordinates of bounding box for the triplet query.
+#' @param required whether the existence of a value for the triple is required or not (defaults to TRUE).
+#'   If set to FALSE, then other triples in the query are returned even if this particular triple is missing)
+#' @param within_box if provided, north-west and south-east coordinates of bounding box for the triple query.
+#' @param within_distance if provided, north-west and south-east coordinates of bounding box for the triple query.
 build_part_body=function(query=NA,subject,verb,object,required=TRUE,
                          within_box=c(NA,NA),within_distance=c(NA,NA)){
 
   part_body=query$body
 
   if(verb %in% c("is","==","%in%")){
-    # if the triplet is not a regular RDF triplet but a statement of the type
+    # if the triple is not a regular RDF triple but a statement of the type
     # subject is any of the objects
     values=paste(as_values(object),collapse="\n")
-    new_triplet=glue::glue("VALUES {{subject}}{\n{{values}}\n}",
+    new_triple=glue::glue("VALUES {{subject}}{\n{{values}}\n}",
                            .open="{{",.close="}}")
   }else{
-    # if the triplet is a regular RDF triplet
-    new_triplet=glue::glue("{subject} {verb} {object}.")
+    # if the triple is a regular RDF triple
+    new_triple=glue::glue("{subject} {verb} {object}.")
   }
 
-  # when arg required=FALSE set triplet as optional
+  # when arg required=FALSE set triple as optional
   if(!required){
-    new_triplet=paste0("OPTIONAL {",new_triplet,"}")
+    new_triple=paste0("OPTIONAL {",new_triple,"}")
   }
   # when arg within_box is provided use service wikibase:box
   if(!is.na(within_box[[1]][1])){
-    new_triplet=paste0("SERVICE wikibase:box {\n",
-                       new_triplet,"\n",
+    new_triple=paste0("SERVICE wikibase:box {\n",
```

Comment:
changing all "triplet" occurrences to "triple"

Reply:
Yep! only realized lately that triplet was french not english...